src/reducers/certificate.js

```diff
@@ -12,6 +12,24 @@ export const initialState = {
   storeError: null,
   storeLoading: false,
 
+  // To remove all the individual checks and messages and instead have one pending status
```

Comment:
Proposing a new state here, comment pls?

Reply:
Intend to have a selector to reduce the 4 states to isVerified. Another selector to reduce the details sections into message to display for errors.